frontend/components/auth/login-form.tsx

```diff
@@ -45,7 +45,7 @@ export function LoginForm() {
       toast({
         title: "Error",
         variant: "destructive",
-        description: "Login Failed.",
+        description: "Username or/and password provided is wrong.",
```

Comment:
I think the possible error codes for the login endpoint are:
* 200
* 400 (missing fields)
* 401 (incorrect email/password)
* 500 (internal server error)

400, 401, 500 will technically be caught by this exception. Might be better to update the login function in `auth-context.tsx` to handle these different error codes.